test/functional/admin/emergency_banner_controller_test.rb

```diff
@@ -11,7 +11,7 @@ class Admin::EmergencyBannerControllerTest < ActionController::TestCase
 
   should_be_an_admin_controller
 
-  %i[show].each do |action_method|
+  %i[edit show update].each do |action_method|
     test "#{action_method} action is not permitted to non-GDS admins" do
       login_as :departmental_editor
       get action_method
```

Comment:
Not the end of the world but does this need to be a PATCH for update? I'm assuming this is hitting the same endpoint as show atm.

Reply:
Ooh, nice spot.  I'll update that.

Reply:
And I'll do the same for the `destroy` action.